### ISSUE TITLE
[BUG] Accessibility - App - Labels must be unique - "See Data Usage" repeats

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#4874]
**Describe the issue**
On the privacy settings screen, "See Data Usage" repeats

_"WCAG Success Criterion 2.4.6: Headings and Labels (Level AA). This criterion emphasizes the importance of providing descriptive headings and labels to help users understand the content and navigate the page effectively1.

Using generic labels like "read more" without additional context can be problematic because they do not provide enough information about the link's destination or purpose. This can be confusing for users, especially those relying on screen readers, as they may encounter multiple "read more" links without knowing what each one refers to.

To avoid this issue, it's recommended to:

Provide unique and descriptive labels: Instead of using "read more," specify what the user will read more about, such as "read more about our services" or "read more about the latest news."
Use ARIA attributes: If it's not possible to change the visible text, you can use ARIA attributes like aria-label to provide additional context for screen readers."_

**To Reproduce**
**Steps to reproduce the behavior:**
1. Launch App
2. Sign in with NetID
3. Go to 'Settings'
4. Click on 'My App Privacy Settings' from the dropdown
5. Select  'Manage and Understanding Your Privacy'
6. Click "Expand All" link to the right under Features and Data collection

**Expected behavior**
Each tabbable element would have a unique ID. 

**Actual behavior**
"See Data Usage" repeats

**Screenshots**

![Image](https://github.com/user-attachments/assets/909b3afb-71a2-44c3-96ba-1b6260a4fb79)

**Testing Environment & Devices (please complete the following information):**
**Device**: Google Pixel 8a
**OS**: Android 15

**Additional context**

**Comment by vburgett** [comment]
@TodorBachvarov to solve this issue please update the text to “See Notifications Data Usage,” “See Sharing Data Usage,” etc.

**Comment by acovher2** [comment]
@vburgett and @TodorBachvarov To be clear this is a bug that is going to affect more that this statement. Anywhere in the code where a button label or link ID is repeated. For instance, every star that just reads "Favorite" will need to have a label that indicates which thing you are favoriting specifically with that button. Anywhere that just reads "Button double tap to activate" like the toggle for Directory Visibility. I cannot open a bug for every instance. This will undoubtedly need developers to CTRL + F in the code to find the reoccurrences and instances of these.  I'm also happy to do a screen share if that helps.

**Comment by acovher2** [comment]
![Image](https://github.com/user-attachments/assets/c815d965-e552-44b8-bca0-32da94d46ec7)

![Image](https://github.com/user-attachments/assets/292667cc-0baa-43d1-8c7f-e4b519aed9e4)

![Image](https://github.com/user-attachments/assets/f87aa6f8-5efe-44ec-8682-e12e5f01d593)

**Comment by acovher2** [comment]
Tagging this with WEB as this pertains to both App and Web instance production versions 7.0.20.

**Comment by acovher2** [comment]
@vburgett Could the labels for these be unique by including the field title in them first? Similar to what we did with the repeated "See data usage" solution? 
i.e. "Pronouns Edit Visibility button" etc

**Comment by TodorBachvarov** [comment]
@acovher2, The title is included in the semantic label for the text field, edit button and edit visibility button. Fixed in version 7.3.34

**Comment by acovher2** [comment]
@TodorBachvarov Test pass 7.3.35

**Comment by acovher2** [comment]
@TodorBachvarov Re-opening this issue with new example:

![Image](https://github.com/user-attachments/assets/08361077-84c6-4a5a-bc47-b4e309216f77)
Please use the title of the card with the delete button (same solve you did for "see more data") to ensure the screen reader user know context of each delete button. 

Thanks

**Comment by TodorBachvarov** [comment]
@acovher2, As the delete button issue was fixed in separate ticket, probably we can close this one.

**Comment by acovher2** [comment]
@TodorBachvarov I intentionally added that Delete button example to this ticket because the resolution is the same. Vanessa and I are not going to list what to label each button. The solve for the delete is to use the title of the card + delete button to uniquely identifiable by assistive technology, 
I will delete the example from #5470 because those unlabeled buttons will need someone to tell you what to label them.

**Comment by acovher2** [comment]
Verified as fixed in 8.0.8
